### PR TITLE
Documentation update: WPT data import; image links fixes

### PR DESCRIPTION
**Pull request by CoolCold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ x ] Documentation content changes

## Other information

#114

### CODE REVIEW

File: docs/docs/getting-started/configuring-your-first-project.md [hunk 1]
```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
...
+
+![Run import task](/docs/static/img/wpt-update.png)
+
+
 ## Creating your first project
 
 To create your first project, head to `<your Falco instance URL>/admin/` and connect to the admin interface. Then click on the `+ Add` button in the `Projects` section, and you should see the following form:
 
-![The form used to create a project](/img/create-project.png)
+![The form used to create a project](/docs/static/img/create-project.png)
```
Comment (phacks) [inline docs/docs/getting-started/configuring-your-first-project.md]: Are you sure about this?

When I try accessing https://getfal.co/img/create-project.png it works as intended, but if I go to https://getfal.co/docs/static/img/create-project.png it gives a Docusaurus 404.

File: docs/docs/getting-started/configuring-your-first-project.md [hunk 2]
```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
 
 This email contains your WebPageTest API Key that we will use in the next section.
 
+## Import data from WPT
+
+WPT service provides numerouse options for running your tests ( location, bandwidth ), so you **must** import this data into your setup. To do this, via admin interface:
+1. head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/`
```
Comment (phacks) [inline docs/docs/getting-started/configuring-your-first-project.md]:
```suggestion
1. Head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/` 
```

File: docs/docs/getting-started/configuring-your-first-project.md [hunk 3]
```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
 
 This email contains your WebPageTest API Key that we will use in the next section.
 
+## Import data from WPT
+
+WPT service provides numerouse options for running your tests ( location, bandwidth ), so you **must** import this data into your setup. To do this, via admin interface:
+1. head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/` 
+1. mark checkbox for `Get all wpt audit configurations: 0 1 * * * (m/h/d/dM/MY) UTC`
```
Comment (phacks) [inline docs/docs/getting-started/configuring-your-first-project.md]:
```suggestion
1. Check the `Get all wpt audit configurations: 0 1 * * * (m/h/d/dM/MY) UTC` checkbox
```

File: docs/docs/getting-started/configuring-your-first-project.md [hunk 4]
```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
 
 This email contains your WebPageTest API Key that we will use in the next section.
 
+## Import data from WPT
+
+WPT service provides numerouse options for running your tests ( location, bandwidth ), so you **must** import this data into your setup. To do this, via admin interface:
+1. head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/` 
+1. mark checkbox for `Get all wpt audit configurations: 0 1 * * * (m/h/d/dM/MY) UTC` 
+1. from combobox (dropdown list) choose to Run the task
```
Comment (phacks) [inline docs/docs/getting-started/configuring-your-first-project.md]:
```suggestion
1. Choose the Run the task option from the dropdown list
```

File: docs/docs/getting-started/configuring-your-first-project.md [hunk 5]
```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
 
 This email contains your WebPageTest API Key that we will use in the next section.
 
+## Import data from WPT
+
+WPT service provides numerouse options for running your tests ( location, bandwidth ), so you **must** import this data into your setup. To do this, via admin interface:
```
Comment (phacks) [inline docs/docs/getting-started/configuring-your-first-project.md]:
```suggestion
WPT provides numerous options for running your tests (location, bandwidth), so you **must** import this data before setting up your first project. You can do this via the admin interface:
```

File: docs/docs/getting-started/configuring-your-first-project.md (hunk 1 above)
Comment (CoolCold) [reply docs/docs/getting-started/configuring-your-first-project.md]: I was checking with Github's preview. Doesn't it mean that dir structure here and at getfal.co differs?

Comment (phacks) [reply docs/docs/getting-started/configuring-your-first-project.md]: My guess is that [Docusaurus](https://v2.docusaurus.io/), the docs framework we use for Falco, doesn’t exactly mimic the URL structure with the file structure with regards to images / static assets.

Comment (CoolCold) [reply docs/docs/getting-started/configuring-your-first-project.md]: well, i cannot check it in advance, so no problem from my side to keep url as it was

Comment (phacks) [reply docs/docs/getting-started/configuring-your-first-project.md]: Great then! Thanks!

File: docs/docs/getting-started/configuring-your-first-project.md [hunk 6]
```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
...
+
+![Run import task](/docs/static/img/wpt-update.png)
+
+
 ## Creating your first project
 
 To create your first project, head to `<your Falco instance URL>/admin/` and connect to the admin interface. Then click on the `+ Add` button in the `Projects` section, and you should see the following form:
 
-![The form used to create a project](/img/create-project.png)
+![The form used to create a project](/docs/static/img/create-project.png)
```
Comment (phacks) [inline docs/docs/getting-started/configuring-your-first-project.md]:
```suggestion
![The form used to create a project](/img/create-project.png)
```

**Comment by phacks** [comment]
Hi @CoolCold, I tried reverting the image path changes in order to merge your PR, but I don’t have write rights on your fork. Could you push a new commit with the image paths as they were?

Thanks!

**Comment by phacks** [comment]
I’m closing this PR in favour of #129 which reverts the image path changes. Thanks again @CoolCold!

**Comment by phacks** [comment]
@all-contributors Please add @CoolCold for doc!

**Pull request closed without merging**